contracts/acl/ACL.sol

```diff
@@ -101,12 +101,12 @@ contract ACL is IACL, AragonApp, ACLHelpers {
     * @param _role Identifier for the group of actions in app given access to perform
     * @param _params Permission parameters
     */
-    function grantPermissionP(address _entity, address _app, bytes32 _role, uint256[] _params)
+    function grantP(address _entity, address _app, bytes32 _role, uint256[] _params)
         onlyPermissionManager(_app, _role)
         public
     {
         bytes32 paramsHash = _params.length > 0 ? _saveParams(_params) : EMPTY_PARAM_HASH;
-        _setPermission(_entity, _app, _role, paramsHash);
+        _set(_entity, _app, _role, paramsHash);
```

Comment:
I wouldn't rename this, as `set` is too generic

Reply:
Changed back to ```_setPermission```.